sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveShim.scala

```diff
@@ -540,29 +442,25 @@ private[client] class Shim_v2_0 extends Shim with Logging {
           logWarning("Caught Hive MetaException attempting to get partition metadata by " +
```

Comment:
this is an existing wrong matching rule, which affects 3.3.0 and above, handling in SPARK-45389 (https://github.com/apache/spark/pull/43191).